documentation/cdi/tutorial-cdi-instantiated-beans.asciidoc

```diff
@@ -1,36 +1,25 @@
 ---
-title: Components instantiated by the framework
+title: Using CDI Beans in Instantiated Components
```

Comment:
Not sure if this goes to right direction. The components instantiated by the framework becomes CDI beans when one starts to use Vaadin CDI. Then, when the components are CDI beans, one can "inject" other CDI beans in them and use other CDI features in them.

Reply:
What about "Using Instantiated Vaadin Components in CDI / Using Vaadin Components in CDI / Using Components in CDI"?

Reply:
It is Framework indeed. And actually all "components" here are not UI components. We should probably talk about objects. 😥

I think I'd go simply with "CDI beans in Vaadin applications"

Reply:
Done